qutebrowser/components/misccommands.py

```diff
@@ -223,41 +223,81 @@ def _insert_text_cb(elem: Optional[apitypes.WebElement]) -> None:
     tab.elements.find_focused(_insert_text_cb)
 
 
+def _wrap_find_at_pos(value: str, tab: apitypes.Tab,
+                      callback: Callable[[Optional[apitypes.WebElement]], None]
+                      ) -> None:
+    try:
+        point = utils.parse_point(value)
+    except ValueError as e:
+        message.error(f"'{value}' is not a valid point: {e}")
+        return
+    tab.elements.find_at_pos(point, callback)
+
+
 @cmdutils.register()
 @cmdutils.argument('tab', value=cmdutils.Value.cur_tab)
-@cmdutils.argument('filter_', choices=['id'])
-def click_element(tab: apitypes.Tab, filter_: str, value: str, *,
+@cmdutils.argument('filter_', choices=['id', 'css', 'position', 'focused'])
+def click_element(tab: apitypes.Tab, filter_: str, value: str = None, *,
                   target: apitypes.ClickTarget =
                   apitypes.ClickTarget.normal,
-                  force_event: bool = False) -> None:
+                  force_event: bool = False,
+                  select_first: bool = False) -> None:
     """Click the element matching the given filter.
 
     The given filter needs to result in exactly one element, otherwise, an
     error is shown.
 
     Args:
         filter_: How to filter the elements.
-                 id: Get an element based on its ID.
-        value: The value to filter for.
+
+            - id: Get an element based on its ID.
+            - css: Filter by a CSS selector.
+            - position: Click the element at specified position.
+               Specify `value` as 'x,y'.
+            - focused: Click the currently focused element.
+        value: The value to filter for. Optional for 'focused' filter.
         target: How to open the clicked element (normal/tab/tab-bg/window).
         force_event: Force generating a fake click event.
+        select_first: Select first matching element if there are multiple.
     """
-    def single_cb(elem: Optional[apitypes.WebElement]) -> None:
-        """Click a single element."""
-        if elem is None:
-            message.error("No element found with id {}!".format(value))
-            return
+    def do_click(elem: apitypes.WebElement) -> None:
         try:
             elem.click(target, force_event=force_event)
         except apitypes.WebElemError as e:
             message.error(str(e))
+
+    def single_cb(elem: Optional[apitypes.WebElement]) -> None:
+        """Click a single element."""
+        if elem is None:
+            message.error(f"No element found matching {filter_}={value}!")
```

Comment:
Maybe add a dict above with the message after "No element found" and "Multiple elements found"? That way, we could have a better error message than "matching focused=None" or so, e.g.:

- "No element found with ID '{value}'"
- "No element found with CSS selector '{value}'"
- "No element found with focus"
- "No element found at position {value}"